src/App.tsx

```diff
@@ -15,71 +15,104 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string,
+  selected: boolean,
+  symbols: { [key: string]: string };
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+    selected: true,
+    symbols: {
+      Jam: '-',
+    },
```

Comment:
Do you really need all these properties in state? 
Actually, it's enough to store only selectedGood there
![image](https://github.com/mate-academy/react_goods-selector/assets/96409563/0857a5d8-983d-4aea-a1f2-f03412a3371c)